mono/metadata/locales.c

```diff
@@ -44,6 +44,19 @@
 
 #undef DEBUG
 
+#define mono_handle_setval(handle,field,value) MONO_HANDLE_RAW (handle)->field = (value)
```

Comment:
Maybe parens around the overall macro though? Expensive to reset CI though. Punt to another PR?

Reply:
I believe you can at least partly enforce the comment by taking the address of `value`.
That limits what forms `value` can take, but includes the forms you want.

`#define mono_handle_setval(handle,field,value) (MONO_HANDLE_RAW (handle)->field = &(value))`